nailgun/entities.py

```diff
@@ -650,20 +650,22 @@ class DiscoveryRule(
         EntityReadMixin,
         EntitySearchMixin,
         EntityUpdateMixin):
-    """A representation of a Foreman Discovery Rule entity."""
+    """A representation of a Foreman Discovery Rule entity.
+
+    .. NOTE:: The ``search_`` field is named as such due to a naming conflict
+        with :meth:`nailgun.entity_mixins.Entity.path`.
+
+    """
 
     def __init__(self, server_config=None, **kwargs):
         self._fields = {
```

Comment:
I've ordered these fields alphabetically, so this change looks bigger than it actaully is.